python/foglamp/common/storage_client/payload_builder.py

```diff
@@ -560,6 +560,50 @@ def JSON_PROPERTY(cls, *args):
                     cls.query_payload["json_properties"] = [json_property]
         return cls
 
+    @classmethod
+    def TIMEBUCKET(cls, timestamp, size="1", fmt=None, alias=None):
```

Comment:
will it be a string? `size="1"`

Reply:
@amarendra-dianomic as per your questions...

I don't believe the size default needs to be dealt with in the payload builder. If there is no size in the payload the storage layer gets it will use the default value of 1.

In retrospect the size would probably be best specified as a number rather than a string. Currently the storage layer only accepts a string. I suggest we leave it as a string for now and look at make it accept a number later.

I think seconds is good enough for this, the idea was to have time bucket style readings, for sub-second values to make sense I think we would need to have a very high frequency sensor. Again let's leave it expressed in seconds until we see a need for sub-second resolution.